config/crd/bases/identityconfig.identitatem.io_authrealms.yaml

```diff
@@ -615,11 +615,17 @@ spec:
                     type: string
                 type: object
               remediateAction:
+                allOf:
```

Comment:
Wondering why there are 2 enums ?

Reply:
Excelent catch, it seems before the policy project didn't defined the enum for RemediateAction and so we added. Now they do and as we do too we end up with a duplicate. Fixed.
We don't use that field for the time being but we will need to remember (I added a comment in the authorization_type) that it can take capilized and non capitalized string (Inform vs inform)